### PR TITLE
Awesome Books Project

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Pair-programming project

- Add dynamic collection for books ✔️ 
- Use objects for the collection of books ✔️ 
- Add LocalStorage for books ✔️ 
- Add add and remove btns for books ✔
a4c75869d223aa90129ab6c89646ac34
️

### CODE REVIEW

File: html folder/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <main>
+        <section>
+            <h1>Awesome Books</h1>
+            <ul id="books">
+                <!-- <li>
+                    <p>Book 1</p>
+                    <p>Author 1</p>
+                    <button>Remove</button>
+                </li>
+           -->
```
Comment (vechicin) [inline html folder/index.html]: Please remove commented code to keep a clean codebase.

File: html folder/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books Project</title>
+    <link rel="stylesheet" href="/css folder/index.css">
```
Comment (vechicin) [inline html folder/index.html]: You're not linking your CSS file correctly, please fix this.

File: html folder/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <section id="newBookSection">
+            <input id="titleId" type="text" placeholder="Title">
+            <input id="authorId" type="text" placeholder="Author">
+            <button id="addBtn">Add</button>
+        </section>
+    </main>
+    <footer>
+
+    </footer>
+<script src="/js folder/index.js"></script>
```
Comment (vechicin) [inline html folder/index.html]: You're not linking your JavaScript file correctly, please fix this.

File: js folder/index.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+// ----------------------- EVENT LISTENER ------------------- //
+
+addBtn.addEventListener('click', () => {
+  const titleName = title.value;
+  const authorName = author.value;
+  title.value = '';
+  author.value = '';
+  const addNewBook = new BookConstructor(titleName, authorName);
+  booksData.push(addNewBook);
```
Comment (vechicin) [inline js folder/index.js]: [OPTIONAL] Consider using methods that don't mutate the array.

Screen Shot 2022-06-27 at 17 45 14

File: js folder/index.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  authorP.textContent = booksData[index].author;
+  deleteBtn.textContent = 'remove';
+  // appendChilds
+  bookLi.appendChild(titleP);
+  bookLi.appendChild(authorP);
+  bookLi.appendChild(deleteBtn);
+  bookUl.appendChild(bookLi);
+
+  deleteBtn.addEventListener('click', () => {
+    booksData.splice(index, 1);
```
Comment (vechicin) [inline js folder/index.js]: [OPTIONAL] Consider using methods that don't mutate the array.

Screen Shot 2022-06-27 at 17 46 01

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @VitorGuedesMadeira and @aizjicod ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Clean codebase


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js folder/index.js (hunk 4 above)
Comment (VitorGuedesMadeira) [reply js folder/index.js]: Ok, I'll give it a try. Could you explain why it can be a better idea?

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @VitorGuedesMadeira  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the requested changes
- Great communication

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**